README.md

```diff
@@ -1,2 +1,87 @@
-# dev-team
-Proposal for the Dev Team
+# Student Robotics Dev Team Proposal
+
+This document aims to solve the important issue of development by outlining the formation of a number of smaller sub-groups within Student Robotics:
+- The Development Coordination Team.
```

Comment:
I'm not sure I understand the need for this second-level abstraction? What are their responsibilities that aren't already the scope of the core team? All this does is add an additional level of abstraction (and communications overhead) for  whenever the core team may want to know the status of a project.

Reply:
The Core Team exists to facilitate the competition. It is outside of their scope, and to be honest too much work for them to also have to manage development for future years. This ensures that development doesn't pause between the end of the competition and the start of the next cycle. It also means that we won't end up with a situation like last year when there were volunteers wanting to develop and being told that they can't because the Core Team has other things to do that are more important.

Reply:
I suspect you're right. If not only because it allows compartmentalization in the fact that there will likely be people who want to be on 1 team and not the other. 

If volunteers were told development couldn't happen, that was wrong. Development however was highly unlikely to ship. I think the dev team ensures those stay distinct.